src/azure-cli/azure/cli/command_modules/hdinsight/custom.py

```diff
@@ -15,7 +15,7 @@
 def create_cluster(cmd, client, cluster_name, resource_group_name, cluster_type,
                    location=None, tags=None, no_wait=False, cluster_version='default', cluster_tier=None,
                    cluster_configurations=None, component_version=None,
-                   headnode_size='large', workernode_size='large', zookeepernode_size=None, edgenode_size=None,
```

Comment:
In this way, you will change default value for your CLI parameter, right?

Reply:
> In this way, you will change default value for your CLI parameter, which is breaking. Is it by design?

Yes, we want to this, but I forget it is a breaking change. This change will have any impact on existing scripts, yes but it will change the parameter help info. Can we check in in this release cycle?

Reply:
It is not a good experience for customer to have such breaking change in stable azure cli. Is there strong business justification for such breaking change?

Reply:
> It is not a good experience for customer to have such breaking change in stable azure cli. Is there strong business justification for such breaking change?

Because previous the default value is large, and it will ben converted to some concrete value. As time goes, we will deprecate some vm size this will cause issue. If it is not acceptable for this release, it also makes sense for us. And when is the next release which accepts breaking change?

Reply:
S187 - 05/25/2021 release cycle could accept breaking change. CLI only accepts breaking change twice a year.

Reply:
> S187 - 05/25/2021 release cycle could accept breaking change. CLI only accepts breaking change twice a year.

Hi @yonzhan  Thanks. Then this PR will target  S187, and I will announce there will be breaking change in S186.  You can skip this PR in this S185. Thank you~

Reply:
LGTM.